### ISSUE TITLE
Make the user understand when he's seeing an outdated book version (v1)

### ISSUE THREAD

**Issue by Powderking** [git/git-scm.com#1011]
On this page:
https://git-scm.com/book/de/v1/Git-auf-dem-Server-Einrichten-des-Servers

When setting up the .ssh directory, its permission should be set to 700 otherwise the key authorization will not work properly.

Please change to
```
    $ sudo adduser git
    $ su git
    $ cd
    $ mkdir .ssh
    $ chmod 700 .ssh
```

The same is for authorized_keys which should set to 600.

**Comment by sxlijin** [comment]
Thanks for bringing this up.

This seems to happen because Pro Git v2 has not been translated to German,
which results in the German version pointing to the outdated Pro Git v1.
(Currently, progit2 has been translated to the following: fr, es, ko, sr,
zh, id, ru, nl, ja, uk, zh-tw, uz, it, sl, tr, be.)

Not sure how this should be addressed - perhaps a large overhead CSS banner
warning users that Pro Git v1 is outdated?

**Comment by peff** [comment]
> Not sure how this should be addressed - perhaps a large overhead CSS banner
warning users that Pro Git v1 is outdated?

Yeah, I think that would be a good idea. PR's welcome.

**Comment by jnavila** [comment]
Anyway, this remark holds, even for progit v1  and as long as progit v2 is not translated into german, you can address some PRs to https://github.com/progit/progit

Now, would you interested refreshing the first edition into the second one?

**Comment by pedrorijo91** [comment]
Migrating this issue into an issue about creating the banner on outdated book versions

**Comment by jasonlong** [comment]
Not sure what (if any) decision was made around dropping the v1 book (#1149 via #1140), but probably makes sense to close either this issue or #1149?

**Comment by pedrorijo91** [comment]
closing this one. As pointed by @jnavila  on https://github.com/git/git-scm.com/issues/1140#issuecomment-367491519, progit v1 is probably not accessed with frequency enough to keep it. As such, this issue will be closed, and the problem will be addressed on #1149
